cmd/relay-runtime-tools/runner.go

```diff
@@ -61,13 +64,41 @@ func (rr *realRunner) Run(args ...string) error {
 	signal.Notify(rr.signals)
 	defer signal.Reset()
 
+	logOut, err := postLog(mu, &plspb.LogCreateRequest{
+		Name: "stdout",
+	})
+	if err != nil {
+		log.Println(err)
```

Comment:
We should exit after this, right? Otherwise it will just panic below.

Reply:
That was originally there to ensure we do not fail (or timeout) in the existing integration tests and development environment if the PLS is not configured. But I believe I've added all the handling for that downstream now, so I'll verify that.

Additionally, the idea right now is to continue with the task at hand (no pun intended) even if failures occur during the logging, and to continue to write to stdout/stderr in those cases.

This could use a safety check below, though.

Long term, I'm not entirely sure how we want to handle errors within this code.

Reply:
Obviously we'll need to ensure we handle transient errors properly and thoroughly, but in the case of unrecoverable errors, I don't think the answer it going to be interrupt the task if cannot log the output.

Reply:
Added a bit of necessary error handling based on this comment.